### ISSUE TITLE
multicluster e2e test regression

### ISSUE THREAD

**Issue by howardjohn** [istio/istio#18211]
On 1.4, multicluster passes 6/6 times on 10/22: https://testgrid.k8s.io/istio_release-1.4_istio_postsubmit#pilot-multicluster-e2e_istio_release-1.4_postsubmit&width=20&exclude-non-failed-tests=
On master, multicluster failed 8/11 times on 10/22: https://testgrid.k8s.io/istio_istio_postsubmit#pilot-multicluster-e2e_istio_postsubmit&width=20&exclude-non-failed-tests=

Prior to this, master was passing 100% as well

The first failure was for https://prow.k8s.io/view/gcs/istio-prow/logs/pilot-multicluster-e2e_istio_postsubmit/446 associated with https://github.com/istio/istio/pull/18111. Given that this PR impacts the tests that were failing, and due to timing, I suspect this is the root cause.

**Comment by howardjohn** [comment]
First failing presubmit: https://prow.k8s.io/view/gcs/istio-prow/pr-logs/directory/pilot-multicluster-e2e_istio/2028 at 6:40am, 40 minutes after #18111 was merged, so very likely the root cause

**Comment by howardjohn** [comment]
Actually I think there are three distinct multicluter test regressions all at once:
* #18111 broke TestSidecarScope
* #18030 broke
   Evidence: 
    * the test in release-1.4 postsubmit has never failed, but it fails in presubmit of a backport of #18030: https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_istio/18197/pilot-multicluster-e2e_istio_release-1.4/52
    * First postsubmit failure: https://prow.k8s.io/view/gcs/istio-prow/logs/pilot-multicluster-e2e_istio_postsubmit/454, which is the postsubmit run for #18030
    * First presubmit failures were all from #18030 before it merged, but were retested (https://prow.k8s.io/view/gcs/istio-prow/pr-logs/directory/pilot-multicluster-e2e_istio/1946, https://prow.k8s.io/view/gcs/istio-prow/pr-logs/directory/pilot-multicluster-e2e_istio/1930)
* Finally, there have been some issues with setting up firewall rules. This may be due to https://status.cloud.google.com/incident/cloud-networking/19020

**Comment by incfly** [comment]
I don't doubt this could be due to the auto mTLS PR breaking the multi cluster. I saw it sometimes in the presubmit tests, but succeed later when about to debug it...

Curious how you narrow the root cause though. I'm looking at the https://testgrid.k8s.io/istio_istio_postsubmit#pilot-multicluster-e2e_istio_postsubmit&include-filter-by-regex=.*Ingress.* I saw only one failures for post submit run 454, no more runs or failure after that.

Is my way of filtering test runs wrong?

Wanna to learns some tips of how to do this from testgrid...
